manim/animation/transform.py

```diff
@@ -227,6 +227,30 @@ def interpolate(self, alpha: float) -> None:
 
 
 class ClockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: ClockwiseTransform1
+        :quality: medium
+
+        class ClockwiseTransform1(Scene):
+            def construct(self):
+                self.play(ClockwiseTransform(Square(), Circle()))
```

Comment:
I'm not sure this example shows what `ClockwiseTransform` does particularly well. Can you explain what you would like to illustrate with it?

Reply:
if I used a square instead of a text, it bulges out slightly more while transforming clockwise. I just wanted show manim users what `ClockwiseTransform` can do to shapes

Reply:
I think it's fine to have this example just to show how it looks like.

Reply:
so, should I mark the conversation as resolved?

Reply:
You should let behackl respond.

Reply:
Compare `Transform` and `ClockwiseTransform` side by side:

https://user-images.githubusercontent.com/11851593/152309007-96761772-2f9f-4fa9-98cd-ebca5e3aab23.mp4

I couldn't really tell what `ClockwiseTransform` does by looking at this. The regularity of the shapes (and the fact that the points *almost* move along arcs anyways) makes the difference hard to spot (or at least to understand).

My suggestion would be to change this into an example like
```py
class Clock2(Scene):
    def construct(self):
        dl, dr = Dot(), Dot()
        sl, sr = Square(), Square()

        VGroup(dl, sl).arrange(DOWN).shift(2*LEFT)
        VGroup(dr, sr).arrange(DOWN).shift(2*RIGHT)

        self.add(dl, dr)
        self.wait()
        self.play(
            ClockwiseTransform(dl, sl),
            Transform(dr, sr)
        )
        self.wait()
```
which renders as this:

https://user-images.githubusercontent.com/11851593/152309787-470dd249-25ce-484b-9cba-1dae3b04efb4.mp4